Package.swift

```diff
@@ -462,6 +462,20 @@ let package = Package(
       ]
     ),
 
+    .testTarget(
+      name: "SwiftJavaToolTests",
+      dependencies: [
+        "SwiftJavaTool",
```

Comment:
Actually, I suspect this is causing the trouble here... let's not test the executable tool target but the Lib target; AFAIR depending on executables is rather problematic